card/mod/settings/set/type/setting.rb

```diff
@@ -81,11 +79,7 @@ def duplicate_check rules
   end
 
   view :rule_help do
-    <<-HTML
-      <div class="alert alert-info">
-        #{process_content '{{+*right+*help|content}}'}
-      </div>
-    HTML
+    nest [:all, card.name], view: :rule_help
```

Comment:
Well, this was one fast (and I thought reasonably elegant) way to solve the problem that help text renders incorrectly on settings.  See http://wikirate.org/*read

The issue is that help text is treated as special text, almost more like structure cards. Nests are often rendered in the context of the card that they're helping with, eg http://wikirate.org/*edited+*right+*help

But none of the settings have this kind of help text; it's all basically absolute.  See http://wikirate.org/Setting.  But because of the former situation, help text doesn't render correctly without a context.

So my thought was to choose a simple context that makes sense here (*all).   It immediately fixes a current problem, because I had already solved this problem for rules.

I'll add a little documentation.

Reply:
tests would be good

Reply:
added one (that would fail without this change)